templates/.goreleaser.yml.tpl

```
@@ -29,8 +29,8 @@ builds:
       {{- end }}
     env:
       - CGO_ENABLED={{ stencil.ApplyTemplate "cgoEnabled" | trim }}
-      {{- $cmd :=  $cmdName | split "-" }}
       {{- $blockName := (printf "%vAdditionalEnv" ($cmdName | replace "-" "")) }}
+      {{- $blockName = ($blockName | replace "_" "") }}
```

suggestion: you should be able to merge these lines together.
```suggestion
      {{- $blockName := (printf "%vAdditionalEnv" ($cmdName | replace "-" "" | replace "_" "")) }}
```